### ISSUE TITLE
"Change wallet password" rename to "Change Encryption Password"

### ISSUE THREAD

**Issue by colvano** [Encrypt-S/navpi#6]
Change wallet password is confusing and change encryption covers what it does. This is for the Stakebox UI control panel.

- - -
[Elegantt data. What's this?](http://bit.ly/elegantt-for-trello-whats-this)
[](Elegantt!UNITO-UNDERSCORE!data:dont!UNITO-UNDERSCORE!delete{"ignored":false,"autoPlanned":false,"ownerId":false,"dependencies":[]})

**Comment by Satoshinaire** [comment]
Would "Change Wallet Encryption Password" be clearer, or is that too wordy, @colvano?
